test/pkg/gitea/scm.go

```diff
@@ -277,30 +277,11 @@ func CreateGiteaUserSecondCnx(topts *TestOpts, username, password string) (pgite
 	return secondprovider, newuser, err
 }
 
-func CreateForkPullRequest(t *testing.T, topts *TestOpts, secondcnx pgitea.Provider, accessMode, command string) *gitea.PullRequest {
-	forkuserinfo, _, err := secondcnx.Client.GetMyUserInfo()
-	assert.NilError(t, err)
+func CreateForkPullRequest(t *testing.T, topts *TestOpts, secondcnx pgitea.Provider, accessMode string) *gitea.PullRequest {
 	forkrepo, _, err := secondcnx.Client.CreateFork(topts.Opts.Organization, topts.TargetRefName,
 		gitea.CreateForkOption{})
 	assert.NilError(t, err)
 	topts.ParamsRun.Clients.Log.Infof("Forked repository %s has been created", forkrepo.CloneURL)
-	cloneURL, err := MakeGitCloneURL(forkrepo.CloneURL, forkuserinfo.UserName, secondcnx.Password)
-	assert.NilError(t, err)
-	newopts := &TestOpts{
-		GitCloneURL:   cloneURL,
-		TargetRefName: topts.TargetRefName,
-		ParamsRun:     topts.ParamsRun,
-	}
-	processed, err := payload.ApplyTemplate("testdata/pipelinerun-alt.yaml", map[string]string{
-		"TargetNamespace": topts.TargetNS,
-		"TargetEvent":     topts.TargetEvent,
-		"TargetBranch":    topts.DefaultBranch,
-		"PipelineName":    "pr",
-		"Command":         command,
-	})
-	assert.NilError(t, err)
-	entries := map[string]string{".tekton/pr.yaml": processed}
-	PushFilesToRefGit(t, newopts, entries, topts.TargetRefName)
```

Comment:
Can you remind me the reason of removal :thinking:

Reply:
because when we create PR, we have commit already, this was adding extra commit to the PR